frontend/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,107 @@
+import React,{ useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import { GoogleLoginButton } from 'react-social-login-buttons';
+import { Link,useNavigate } from 'react-router-dom';
+import Form from 'react-bootstrap/Form';
+import './LoginPage.css';
+import axios, { AxiosError,HttpStatusCode } from 'axios';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { AUTH_LOGIN_URL } from '../../constants/constants';
+import { allowCorsForAxios } from '../../utils/allowCorsForAxios';
+import AuthTokens from '../../interfaces/AuthTokens';
+import { saveTokens } from '../../services/auth.service';
+import LoginFormData from '../../interfaces/LoginFormData';
+import isEmailValidCheck from '../../utils/isEmailValidCheck';
+
+
+const LoginPage = () => {
+  const navigate = useNavigate();
+
+  const [loginFormData,setLoginFormData] = useState<LoginFormData>({ email: '', password: '' });
+
+  const handleInputChange = (fieldName:string, value:string) => {
+    setLoginFormData({ ...loginFormData, [fieldName]: value });
+  };
+
+  const isInputValidCheck = (email:string, password:string) => {
+    if(!email || !password) {
+      toast.error('Please fill in all fields');
+      return false;
+    }
+    const isEmailValid = isEmailValidCheck(email);
+    if(!isEmailValid) {
+      toast.error('Please enter a valid email address');
+      return false;
+    }
+    return true;
+  }
+  
+  const handleLoginSubmit =async (e: React.FormEvent<HTMLFormElement>) => {
+    try{
+    e.preventDefault();
+    const { email, password } = loginFormData;
+    const isInputValid = isInputValidCheck(email, password);
+    if(!isInputValid) return;
+    allowCorsForAxios(axios)
+    const tokens = await axios.post(AUTH_LOGIN_URL, {email, password})
```

Comment:
also, it seems that there's an issue with the login process even after successfully creating a new user. The login request returns a 401 Unauthorized status code, indicating that the user authentication failed. it might be that the server doesn't correctly compare the credentials from the frontend page and the user data stored in the backend.

Reply:
will be checked

Reply:
this bug was fixed